pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1116,6 +1149,154 @@ func lookupDomainConfig(ctx *domainContext, key string) *types.DomainConfig {
 	return &config
 }
 
+func runningOnKVM() bool {
```

Comment:
Shouldn't the support be expressed inside the hypervisor interface instead of hard-coding it in domainmgr?

Thus it seems to make sense to add a IsCpuPinningSupported() to the hypervisor interface and have kvm.go return true, and add a TBD comment for Xen where its function returns false.

Reply:
Hmm... The logic of the isolation of the VMs from each other will still work, due to the cgroups/cpuset usage, which is not QEMU-specific. The only part that will not work is - the pinning of a VCPU to a CPU, which _is_ QEMU-specific.
Do we still want to add some extra logic to distinguish the Xen case?

Reply:
Let's worry about Xen later.

Reply:
Ok, that makes sense to me.